legend-engine-config/legend-engine-repl/legend-engine-repl-relational/pom.xml

```diff
@@ -26,6 +26,36 @@
     <artifactId>legend-engine-repl-relational</artifactId>
     <name>Legend Engine - REPL - Relational</name>
 
+    <properties>
+        <skip.yarn>false</skip.yarn>
+        <repl.web-application.version>11.29.0</repl.web-application.version>
+        <repl.web-application.url>https://registry.npmjs.org/@finos/legend-application-repl-deployment/-/legend-application-repl-deployment-${repl.web-application.version}.tgz</repl.web-application.url>
```

Comment:
Is it possible to make the initial part of the URL a Maven property? We have two places where we download dependencies from a NPM repository (here and also Pure IDE Light HTTP server pom). 
If we could extract out https//registry.npmjs.org into a central place then that makes it easier to change the URL when building in a corporate enviroment where registry.npmjs.org may be blocked.